FRCore/FRCore/Keychain/SecuredKey.swift

```diff
@@ -180,17 +180,25 @@ public struct SecuredKey {
     
     /// Decrypts Data object using SecuredKey object
     /// - Parameter data: Decrypted Data object
-    public func decrypt(data: Data) -> Data? {
+    public func decrypt(data: Data, secAlgorithm: SecKeyAlgorithm = .eciesEncryptionCofactorVariableIVX963SHA256AESGCM) -> Data? {
         
-        guard SecKeyIsAlgorithmSupported(privateKey, .decrypt, algorithm) else {
-            Log.e("\(algorithm) is not supported on the device.")
+        guard SecKeyIsAlgorithmSupported(privateKey, .decrypt, secAlgorithm) else {
+            Log.e("\(oldAlgorithm) is not supported on the device.")
```

Comment:
same as above can you confirm is this suppose to be  -  Log.e("\(secAlgorithm) is not supported on the device.") ?

Reply:
good catch